geofileops/util/_geoops_sql.py

```diff
@@ -2309,6 +2320,8 @@ def symmetric_difference(
         # Now we are ready to move the result to the final spot...
         if output_path.exists():
             gfo.remove(output_path)
+        elif not output_path.parent.exists():
+            raise ValueError("symmetric_difference: output_path doesn't exist")
```

Comment:
This should be moved to the top of the function, where there is also a check of output_path exists, otherwise the Exception is still raised after all calculations...

Probably it is better, for consistency reasons, to do the gfo.remove,... also at the top of the function as as is being done in gfo.erase, gfo.union, gfo.identity,...